packages/pds/src/crawlers.ts

```diff
@@ -0,0 +1,37 @@
+import { AtpAgent } from '@atproto/api'
+import { crawlerLogger as log } from './logger'
+import { MINUTE } from '@atproto/common'
+
+const NOTIFY_THRESHOLD = 20 * MINUTE
+
+export class Crawlers {
+  public agents: AtpAgent[]
+  public lastNotified = 0
+
+  constructor(public hostname: string, public crawlers: string[]) {
+    this.agents = crawlers.map((service) => new AtpAgent({ service }))
+  }
+
+  async notifyOfUpdate() {
+    const now = Date.now()
+    if (now - this.lastNotified < NOTIFY_THRESHOLD) {
+      return
+    }
+
+    await Promise.all(
+      this.agents.map(async (agent) => {
+        try {
+          await agent.api.com.atproto.sync.notifyOfUpdate({
+            hostname: this.hostname,
+          })
+        } catch (err) {
+          log.warn(
+            { err, service: agent.service.toString() },
```

Comment:
The `service` key gets picked-up by some logging software (e.g. datadog) as a special name indicating the source service.  If we could rename to something like `crawler`, that might be a good look.  In the repo subscription logs we had a similar thing happening and ended-up changing it to `provider`.